tests/unit/dataSource/filtering/tests.html

```diff
@@ -979,11 +979,11 @@
 				});
 				test("Test function filter with argument fieldExpressionsOnStrings", function () {
 					var fd;
-					ds.filter([], null, null, "ID >= 1 AND ID < 3");
```

Comment:
@MayaKirova Shouldn't the AND logic be preserved and a new test for the OR logic be written?